dask/bag/tests/test_text.py

```diff
@@ -77,6 +93,18 @@ def test_files_per_partition():
 
             assert b.count().compute() == 20, "All 20 lines should be read"
 
+            with pytest.warns(UserWarning):
```

Comment:
Why are we asserting a warning here?

Reply:
This is from line 72(master) / 88(HEAD). This was already present in the old test.